### PR TITLE
Create Innovation-Specialist-GS-15.md

### PR DESCRIPTION
**Pull request by pixelwake**
To #127

### CODE REVIEW

**Comment by fureigh** [comment]
I'll assume this is pending too until you say "Ready for review." :) Three cheers for iterating on process!

**Comment by pixelwake** [comment]
Ready for review.

File: _positions/Innovation-Specialist-GS-15.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Innovation Specialist, GS-0301-15
+permalink: who-we-are-hiring/positions/Innovation-Specialist-GS-15/
```
Comment (fureigh) [inline _positions/Innovation-Specialist-GS-15.md]: See questions and comments on the corresponding section of #128.

File: _positions/Innovation-Specialist-GS-15.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+team: 18F
+active: false
+---
+
+##Innovation Specialist, GS-0301-15
+**GS Grade Level:** 15
+
+This position is located in the General Services Administration (GSA), Office of Citizen Services Innovation and Technology & 18F (OCSIT/18 F). The 18F office is charged with developing innovative digital tools and services. 18F uses lean startup and agile development principles to make easy things easy and hard things possible. As an Innovation Specialist, the incumbent serves the government and makes an impact on a truly massive scale. The position requires that fellows perform work of an exceptional degree of difficulty across a wide range of topics at the convergence of technology, policy, and delivery. The incumbent leverages his/her experience deploying high quality, user centric platforms and services to lead major initiatives, consult on systems and policy proposals, and provide technical, policy, and programmatic guidance to government. The position involves access to PII data.
```
Comment (fureigh) [inline _positions/Innovation-Specialist-GS-15.md]: 'Incumbent' is used here for the first time, and 'fellow' appears below without previous mention of a fellowship. Can we standardize to something else? This question also pertains to #150 and #128.

File: _positions/Innovation-Specialist-GS-15.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+- Analyzing and scoping the technical and policy requirements needed to implement complex digital solutions;
+- Assessing the state of digital services provided by the Federal Government, and working to align strategies and practice; conferring with key government/private officials and top experts in the field, representing the agency at technical symposia and/or conferences;
+- Developing authoritative papers/reports; developing and leading adoption of government-wide standards for digital services, leading to their adoption across citizen and business-facing government services;
+- Demonstrating thought leadership and contributing best practices in a variety of technologies and principles;
+- Contributing to a talent strategy that includes recruiting innovators and entrepreneurs to participate in solving complex and esoteric challenges; and
+- Performing other duties, as assigned.
+
+###Qualifications Required:
```
Comment (fureigh) [inline _positions/Innovation-Specialist-GS-15.md]: 'Required' --> 'required'

File: _positions/Innovation-Specialist-GS-15.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+###Major duties:
+Typical duties for the fellow include but are not limited to:
+
+- Conceiving/recommending projects/studies to advance the state of the art in the specialty area; applying forward-thinking design and development principles to product or services development, delivery and program management, such as lean startup, user-centered design, and agile development;
```
Comment (fureigh) [inline _positions/Innovation-Specialist-GS-15.md]: Flagging this for incorporation of changes from #128, provided they're approved there.

File: _positions/Innovation-Specialist-GS-15.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: Innovation Specialist, GS-0301-15
+permalink: who-we-are-hiring/positions/innovation-specialist-gs-15/
+team: 18F
```
Comment (pixelwake) [inline _positions/Innovation-Specialist-GS-15.md:4]: Fixed permalink case.

File: _positions/Innovation-Specialist-GS-15.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- Analyzing and scoping the technical and policy requirements needed to implement complex digital solutions
+- Assessing the state of digital services provided by the Federal Government, and working to align strategies and practice; conferring with key government/private officials and top experts in the field, representing the agency at technical symposia and/or conferences
+- Developing authoritative papers/reports; developing and leading adoption of government-wide standards for digital services, leading to their adoption across citizen and business-facing government services
+- Demonstrating thought leadership and contributing best practices in a variety of technologies and principles
+- Contributing to a talent strategy that includes recruiting innovators and entrepreneurs to participate in solving complex and esoteric challenges
+- Performing other duties, as assigned
+
+###Qualifications required:
+This highly competitive program seeks talented, diverse individuals from the innovation community, and candidates are required to demonstrate expertise in a number of competencies. The position requires diverse knowledge, including:
```
Comment (pixelwake) [inline _positions/Innovation-Specialist-GS-15.md:26]: Standardized headings

File: _positions/Innovation-Specialist-GS-15.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- Expert analytical skills to apply a wide range of qualitative and/or quantitative methods for assessing and improving program effectiveness
+
+Candidates are also evaluated for their competencies in key areas, including:
+- Analysis and Creative Problem Solving: Expertise determining user needs or identifying market opportunities and designing and deploying solutions
+- Technology acumen: Knowledge of standard and emergent web technologies
+- Entrepreneurship and product management: experience delivering products or services from concept to market
+- Approach and methodology: Expertise applying frameworks that emphasize delivery, iteration, quantitative and qualitative learnings, and responsiveness
+- Empathy and emotional intelligence: Experience in clientfacing roles, navigating bureaucracy, business development, or delivering services with a strong userfacing or consumeroriented element
+- Tools and execution: Experience in design, development and deployment of solutions leveraging current and emergent technologies and best practices
```
Comment (pixelwake) [inline _positions/Innovation-Specialist-GS-15.md:42]: Fixed casing. Deleted and re-added hyphens.

File: _positions/Innovation-Specialist-GS-15.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+- Expert analytical skills to apply a wide range of qualitative and/or quantitative methods for assessing and improving program effectiveness
+
+Candidates are also evaluated for their competencies in key areas, including:
+- Analysis and Creative Problem Solving: Expertise determining user needs or identifying market opportunities and designing and deploying solutions
+- Technology acumen: Knowledge of standard and emergent web technologies
+- Entrepreneurship and product management: experience delivering products or services from concept to market
+- Approach and methodology: Expertise applying frameworks that emphasize delivery, iteration, quantitative and qualitative learnings, and responsiveness
+- Empathy and emotional intelligence: Experience in client facing roles, navigating bureaucracy, business development, or delivering services with a strong user facing or consumer oriented element
+- Tools and execution: Experience in design, development and deployment of solutions leveraging current and emergent technologies and best practices
```
Comment (pixelwake) [inline _positions/Innovation-Specialist-GS-15.md:42]: Fixed casing and deleted/re-added hyphens.

**Comment by pixelwake** [comment]
Final changes made, merging.

**Pull request merged**